x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/custom_query_rule.cy.ts

```diff
@@ -119,11 +121,12 @@ describe('Custom query rules', { tags: [tag.ESS, tag.BROKEN_IN_SERVERLESS] }, ()
     cleanKibana();
```

Comment:
NIT: As we are deleting alerts and rules before each scenario we could delete this clean method :)